src/SchemaForm.vue

```diff
@@ -58,29 +67,20 @@ export default {
       provide('parentSchemaExists', true)
     }
 
-    const { getID } = useUniqueID()
-
-    const parsedSchema = computed(() => {
-      const arraySchema = Array.isArray(props.schema)
-        ? props.schema
-        : Object.keys(props.schema).map(model => ({
-          ...props.schema[model],
-          model
-        }))
-
-      return arraySchema.map(field => ({
-        ...field,
-        uuid: getID(field.model)
-      }))
-    })
+    const { parsedSchema } = useParsedSchema(props)
 
     watch(parsedSchema,
       (schema, oldSchema) => {
```

Comment:
Could we extract this callback too a named function that describes what it does?